### ISSUE TITLE
Trailer vehicles unbalanced due to lack of wheels.

### ISSUE THREAD

**Issue by emptytriangle** [CleverRaven/Cataclysm-DDA#45701]
# Describe the bug
afs_gas_tanker and afs_water_tanker are set up like tow hitch or semi trailers, with only two wheels and getting their third point of support from the tow truck. This counts as an unbalanced vehicle, so if you attach a heavy duty tow cable to the trailer you can't pull it since the trailer is dragging on the ground.

# Steps To Reproduce
Connect a tow vehicle to an afs_gas_tanker or afs_water_tanker and try to pull it. Note the tanker is dragging on the ground.

# Expected behavior
Connecting a tow cable allows one to tow afs_gas_tanker and afs_water_tanker.

# Additional context
I think the easiest way to enable this behavior would be to make the tow hook item (the vehicle part, the one that is for the slave vehicle) count as a wheel for purposes of checking balance.

**Comment by natsirt721** [comment]
Adding a gooseneck connector and a fifth wheel part is what is needed to make towing semi trailers practical. Your description indicates exactly what the issue is, but I don't think that meddling with the tow cable is the solution.

**Comment by emptytriangle** [comment]
I don't really agree or disagree, it's just that there's more that comes along with adding another wheel to the trailer. That's not on a real trailer irl, they don't roll on a wheel there. And that wheel will become a critical component, i.e. if it breaks the whole trailer can no longer be towed

**Comment by jtgibson01** [comment]
@emptytriangle I'm a little late (got here because of the recent mainline commit, which I'm interested in because... reasons), but a "[fifth wheel](https://en.wikipedia.org/wiki/Fifth-wheel_coupling)" is a trailer hitch receptacle, not an actual wheel -- the little horseshoe-shaped slot on the back of a semi-truck is called the fifth wheel because on old carriages, they literally used a horizontal wheel/axle on top of the front axle to allow the entire front axle to rotate (rather than the wheels turning left or right independently like modern steering racks).

In other words, natsirt721 is talking about the need to implement tractor-trailer combinations properly with a proper overlapping trailer hitch, without resorting to the technically-impressive-but-kinda-janky towing system.  Of course, everyone fully recognises the kind of undertaking a proper tractor-trailer implementation is, so s/he is simply stating that the current lack of implementation is probably better than a hacky, half-functional implementation would be.

**Comment by Mom-Bun** [comment]
Gas tanker and Water tanker in aftershock was removed in #47279 fixing this issue.
